src/admin-views/editor/fieldset/advanced.php

```diff
@@ -121,15 +121,15 @@ class="tribe-timepicker tribe-field-end_time ticket_field"
 				data-step="<?php echo esc_attr( $timepicker_step ); ?>"
 				data-round="<?php echo esc_attr( $timepicker_round ); ?>"
 				value="<?php echo esc_attr( $ticket ? $ticket->end_time : null ); ?>"
-				aria-label="<?php esc_html_e( 'Ticket end date', 'event-tickets' ); ?>"
+				aria-label="<?php printf( esc_html__( '%s end date', 'event-tickets' ), tribe_get_ticket_label_singular( 'input_end_time_aria_label' ) ); ?>"
```

Comment:
This only escapes the __ string, not the strings you are inserting via printf(). Can you follow the other examples where I've made suggestions that does it like `echo esc_html( sprintf( .... ) );` here elsewhere in your PR?

Reply:
Also this should be using esc_attr() instead of esc_html()